app/public/components/chips/linkChip.js

```diff
@@ -20,12 +20,12 @@ import { h } from '/js/src/index.js';
  * @param {Navigation} navigation the content of the link displayed to the user
  * @param {string} label displayed chip name
  * @param {string} href the absolute URL targeted by the link
- *
+ * @param {string} additionDecorations string of dot separated css classes like .class1.class2 ...
  * @return {vnode} the linkChip component
  */
 
-export default function linkChip(navigation, label, href) {
-    return h('a.btn.chip.m1.no-text-decoration', {
+export default function linkChip(navigation, label, href, additionDecorations = '') {
```

Comment:
Why decoration and not "classes"? 
But then when we have such thing I prefer to take `string|string[]` in case at some point we change the syntax or use something with another syntax, and adapt it to your needs in the components (in this case, using join('.'))